src/content/docs/synthetics/synthetic-monitoring/getting-started/new-runtime.mdx

```diff
@@ -0,0 +1,100 @@
+---
+title: Convert to the new synthetic runtimes
+tags:
+  - Synthetics
+  - Synthetic monitoring
+  - Runtime conversion
+metaDescription: "Synthetic monitoring's new runtime requires that you make some changes to your already existing monitors."
+---
+
+We're introducing a new runtime for public monitoring locations. We plan to apply the new runtime to private monitors by the end of 2022. While we're providing backwards support for monitors using the Legacy runtime and its features, we recommend converting to the current runtime as soon as possible. 
+
+Going forward, new monitors are defaulting to the new runtime while monitors created using the Legacy version will need additional configuration.
+
+<Callout variant="important">
+Before switching to the new runtime, confirm that you've updated your allow lists to accept network requests from the new runtime’s [public IP ranges](/docs/synthetics/synthetic-monitoring/administration/new-synthetic-public-minion-ips).
+</Callout>
+
+## Benefits [#benefits]
+
+Converting to the new runtime gives you: 
+
+- **Access to newer versions of Google Chrome.** With planned updates twice a year, the new runtime allows us to keep Chrome up to date so your monitors use the most recent Chrome version.   
```

Comment:
Think about removing redundancy in this bullets. For example, is there value in expanding on the bolded text here? Or is there a way to expand the bolded text so that it says everything you need to say, without the following sentence?

Also, we use `*` for bullets, instead of `-`.

Reply:
Also, we don't tend to use bolded text this way. I'm not saying don't do it, because I see value in experimentation :) but you may want to check out how some other docs are using bullets.